packages/e2e/cypress/e2e/common/about.cy.js

```diff
@@ -57,5 +57,7 @@ describe('About page', () => {
 
     cy.contains('#tkn--about--triggers-tile', triggersNamespace);
     cy.contains('#tkn--about--triggers-tile', triggersVersion);
+
+    expect(true).to.eq(false);
```

Comment:
🤔?

Reply:
Sorry, meant to mark this PR as WIP. I was testing a change for the Z (s390x) nightly tests. I needed a failing test to ensure the artifacts still got uploaded correctly.

Should be good to go now though.